src/atwho.js

```diff
@@ -131,7 +131,16 @@ define([
                                                        options.property);
                             }
                             return value;
-                        }
+                        },
+                        afterMatchFailed: function(at, $el) {
+                            if (options.useRichText) {
+                                var content = $el.html().trim();
+                                if (richText.CASE_REF_REGEX.test(content) || form.isValidHashtag(content)) {
```

Comment:
I think `CASE_REF_REGEX` should include a something after the case.

typing #case/ causes the #case/ to be erased on space